docs/docs/configuration.mdx

```diff
@@ -2,31 +2,81 @@
+
+## Secrets Configuration
+
+| Name, shorthand    | Environment variable       | Default    | Description                                                                                                                                                                                                                                                                                                                                                               |
+|--------------------|----------------------------|------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| --secrets-manager  | APPROZIUM_SECRETS_MANAGER  |            | Supported options are "vault" (Hashicorp Vault), "asm" (AWS Secrets Manager), and "local" (Local YAML file)                                                                                                                                                                                                                                                                   |
+| --vault-token-path | APPROZIUM_VAULT_TOKEN_PATH |            | Optional, if set it will cause the latest Vault token to always be pulled from the given file. This option takes precedence over --vaulttoken                                                                                                                                                                                                                             |
+| --vault-token      | APPROZIUM_VAULT_TOKEN      |            | Optional, if set it will be used at the Vault token.                                                                                                                                                                                                                                                                                                                      |
+| --vault-addr       | APPROZIUM_VAULT_ADDR       |            | Required if using Vault as the secrets manaager.                                                                                                                                                                                                                                                                                                                          |
```

Comment:
typo in description

Reply:
https://github.com/cyralinc/approzium/pull/181/commits/2fca5a7d41af828adae7ab530a252fb503acdc5d

Typos are to you what cucumbers are to this cat!

![cat laser eyes](https://media3.giphy.com/media/3oEduQAsYcJKQH2XsI/giphy.gif?cid=ecf05e47b4dqlbvviuq0ueirfvwq87zq6f6lp6x1f20jz9yq&rid=giphy.gif)